taskiq/cli/scheduler/run.py

```diff
@@ -85,14 +85,18 @@ def get_task_delay(task: ScheduledTask) -> Optional[int]:
     if task.cron is not None:
         # If user specified cron offset we apply it.
         # If it's timedelta, we simply add the delta to current time.
+        additional_seconds = 0
         if task.cron_offset and isinstance(task.cron_offset, timedelta):
             now += task.cron_offset
+            additional_seconds += task.cron_offset.seconds + (
```

Comment:
Here you use seconds, which might produce unexpected results. 

```suggestion
            additional_seconds += task.cron_offset.total_seconds() + (
```

In this example if you specify minutes and hours it will also correctly work. Or was it implemented like so on purpose?

Reply:
There was an even worse mistake here, as you can see. The idea was that additional_seconds is the remainder of the seconds from the entire cron_offset. Now fixed it.